patches/chrome-browser-ui-android-omnibox-java-src-org-chromium-chrome-browser-omnibox-suggestions-DropdownItemViewInfoListBuilder.java.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder.java b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder.java
+index 208580a0d53c8a578de68b0f25dd0f3ecf823e05..4f963f68b0d2ed8178fd399904d6db6ff49bde2f 100644
+--- a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder.java
++++ b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder.java
+@@ -92,7 +92,7 @@ class DropdownItemViewInfoListBuilder {
+                 () -> mShareDelegateSupplier == null ? null : mShareDelegateSupplier.get();
+ 
+         mHeaderProcessor = new HeaderProcessor(context, host, delegate);
+-        registerSuggestionProcessor(new EditUrlSuggestionProcessor(
++        registerSuggestionProcessor(new new org.chromium.chrome.browser.omnibox.suggestions.editurl.BraveEditUrlSuggestionProcessor(
```

Comment:
`new new`
also looks like something we could use asm to do

Reply:
Yes, it was fixed later https://github.com/samartnik/brave-core/commit/d77d5d8a5efb87577903438f4eaecac124272694